scripts/screens/clan_creation_screens.py

```diff
@@ -306,6 +370,44 @@ def handle_choose_background_event(self, event):
         elif event.ui_element == self.elements['done_button']:
             self.save_clan()
             self.open_clan_saved_screen()
+    
+    def handle_choose_background_key(self, event):
```

Comment:
we really dont have a list of possible biomes? with it hardcoded in like this, I can see this getting forgotten about and causing problems when more biome are added